stdlib/asm/math/u64.masm

```diff
@@ -168,3 +233,18 @@ export.div_unsafe
     movup.3
     assert.eq           # quotient remains on the stack #
 end
+
+# ===== HELPER FUNCTIONS ======================================================================== #
+
+# Asserts that both values at the top of the stack are u64 values. #
+# The input values are assumed to be represented using 32 bit limbs, fails if they are not. #
+export.u64assert2
+    u32assert
+    movup.3
+    u32assert
+    movup.3
+    u32assert
+    movup.3
+    u32assert
+    movup.3
+end
```

Comment:
A couple of notes on this:

1. Since this is a helper procedure, we don't need to export it. So, we can declare it simply as `proc.u64assert2`
2. I'd move the helper procedures to the top of the file.
3. Maybe rename to `u32assert4` since this is more descriptive of what the procedure actually dose?